extensions/resource-deployment/src/ui/deployClusterWizard/pages/deploymentProfilePage.ts

```diff
@@ -15,28 +15,39 @@ const localize = nls.loadMessageBundle();
 
 export class DeploymentProfilePage extends WizardPageBase<DeployClusterWizard> {
 
-	private _cards: azdata.CardComponent[] = [];
-	private _cardContainer: azdata.FlexContainer | undefined;
+	private _profiles: BigDataClusterDeploymentProfile[] = [];
+	private _cardContainer: azdata.RadioCardGroupComponent | undefined;
 	private _loadingComponent: azdata.LoadingComponent | undefined;
-	private _view: azdata.ModelView | undefined;
 
 	constructor(wizard: DeployClusterWizard) {
-		super(localize('deployCluster.summaryPageTitle', "Deployment configuration template"),
-			localize('deployCluster.summaryPageDescription', "Select the target configuration template"), wizard);
+		super(localize('deployCluster.summaryPageTitle', "Deployment configuration profile"),
+			localize('deployCluster.summaryPageDescription', "Select the target configuration profile"), wizard);
 	}
 
 	public initialize(): void {
 		this.pageObject.registerContent((view: azdata.ModelView) => {
-			this._view = view;
-			this._cardContainer = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'row', flexWrap: 'wrap' }).component();
+			this._cardContainer = view.modelBuilder.radioCardGroup().withProperties<azdata.RadioCardGroupComponentProperties>({
+				cards: [],
+				cardWidth: '240px',
+				cardHeight: '340px',
```

Comment:
was the card height changed on purpose

Reply:
yes, give it more space for the bottom margin